src/EditorFeatures/Core/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -30,73 +30,47 @@ private partial class SymbolInlineRenameInfo : IInlineRenameInfo
         {
             private const string AttributeSuffix = "Attribute";
 
-            private readonly Document _document;
+            private readonly SymbolicRenameInfo _info;
```

Comment:
several of hte fields of this type got extracted into this new type, which is defined/created at the feature-layer.